hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -83,9 +86,12 @@ cdef class PureMarketMakingStrategy(StrategyBase):
                     bid_order_level_spreads: List[Decimal] = None,
                     ask_order_level_spreads: List[Decimal] = None,
                     should_wait_order_cancel_confirmation = True,
+                    moving_price_band: MovingPriceBand = None
```

Comment:
Missing type hint for the parameter

Reply:
Not sure if you are referring to the line above what I changed. I added type hint bool for it.

Reply:
Hello @Protechris. Since the new parameter `moving_price_band` has a default None value, the typehint for it should be `Optional[MovingPriceBand]`

Reply:
Understood. I added the typehint.